File: app/src/main/java/dev/danielholmberg/improve/Models/VipImage.java
package dev.danielholmberg.improve.Models;

import com.google.firebase.database.Exclude;
import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class VipImage {

    private String id;
    private String originalFilePath;

    public VipImage() {}

    public VipImage(String id) {
        this.id = id;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    @Exclude
    public void setOriginalFilePath(String originalFilePath) {
        this.originalFilePath = originalFilePath;
    }

    @Exclude
    public String getOriginalFilePath() {
        return this.originalFilePath;
    }
}

File: app/src/main/java/dev/danielholmberg/improve/legacy/Models/VipImage.kt
package dev.danielholmberg.improve.legacy.Models

import com.google.firebase.database.IgnoreExtraProperties
import com.google.firebase.database.Exclude

@IgnoreExtraProperties
class VipImage(var id: String?) {

    @get:Exclude
    @set:Exclude
    var originalFilePath: String? = null

}